front/src/components/visualize/MapSet/MapSet.vue

```diff
@@ -249,3 +249,15 @@ export default Vue.extend({
 
 })
 </script>
+
+<style scoped>
+.container-button {
+  text-align: center
+}
+
+@media print {
+  .container-button {
```

Comment:
Another tip is to use a special class, eg `no-print`, to use in every place you don't want to see when printing.
On tailwind, they use a specific prefix, `print:`, allowing us to use a `print:hidden` for hiding some HTML. 
See https://tailwindcss.com/docs/hover-focus-and-other-states#print-styles .
For our future refacto ;-)